src/extension.ts

```diff
@@ -2,31 +2,50 @@ import * as vscode from 'vscode';
 import * as k8s from 'vscode-kubernetes-tools-api';
 import * as azcs from 'azure-arm-containerservice';  // deprecated, but @azure/arm-containerservice doesn't play nicely with AzureAccount, so...
 
-import { AKSTreeProvider, AKSClusterTreeNode } from './aks-tree';
```

Comment:
>./aks-tree [](start = 53, length = 10)

Do we still need this aks-tree.ts file, or could we remove it as well?

Reply:
Yes, I had intended to do that and forgotten; thanks for catching that!